web-client/integration-tests/judgeActivityReportJourney.test.ts

```diff
@@ -30,7 +30,7 @@ describe('Judge activity report journey', () => {
     await cerebralTest.runSequence('gotoJudgeActivityReportSequence');
 
     const { isFormPristine, reportHeader } = runCompute(
-      judgeActivityReportHelper as any,
+      judgeActivityReportHelper,
```

Comment:
is this here intentionally?

Reply:
@jasonfu9966 Yes. Change was made to remove type error.